scripts/hist_importer.py

```diff
@@ -0,0 +1,158 @@
+                   "previous browser and import them into qutebrowser.")
+    epilog = ("Databases:\n\tQutebrowser: Is named 'history.sqlite' and can be"
+              " found at your --basedir. In order to find where your basedir"
+              " is you can run ':open qute:version' inside qutebrowser."
+              "\n\tFirerox: Is named 'places.sqlite', and can be found at your"
+              "system\"s profile folder. Check this link for where it is locat"
+              "ed: http://kb.mozillazine.org/Profile_folder"
+              "\n\tChrome: Is named 'History', and can be found at the respec"
+              "tive User Data Directory. Check this link for where it is locat"
+              "ed: https://chromium.googlesource.com/chromium/src/+/master/"
+              "docs/user_data_dir.md\n\n"
+              "Example: hist_importer.py -b firefox -s /Firefox/Profile/"
+              "places.sqlite -d /qutebrowser/data/history.sqlite")
+    parser = argparse.ArgumentParser(
+        description=description, epilog=epilog,
+        formatter_class=argparse.RawTextHelpFormatter
+    )
+    parser.add_argument('-b', '--browser', dest='browser', required=True,
+                        type=str, help='Browsers: {firefox, chrome}')
+    parser.add_argument('-s', '--source', dest='source', required=True,
+                        type=str, help='Source: Full path to the sqlite data'
+                        'base file from the source browser.')
+    parser.add_argument('-d', '--dest', dest='dest', required=True, type=str,
+                        help='Destination: The full path to the qutebrowser '
+                        'sqlite database')
+    return parser.parse_args()
+
+
+def open_db(data_base):
+    """Open connection with database."""
+    if os.path.isfile(data_base):
+        conn = sqlite3.connect(data_base)
+        return conn
+    else:
+        raise sys.exit('\nDataBaseNotFound: There was some error trying to to'
+                       ' connect with the [{}] database. Verify if the'
+                       ' filepath is correct or is being used.'
+                       .format(data_base))
+
+
+def extract(source, query):
+    """Get records from source database.
+
+    Args:
+        source: File path to the source database where we want to extract the
+        data from.
+        query: The query string to be executed in order to retrieve relevant
+        attributes as (datetime, url, time) from the source database according
+        to the browser chosen.
+    """
+    try:
+        conn = open_db(source)
+        cursor = conn.cursor()
+        cursor.execute(query)
+        history = cursor.fetchall()
+        conn.close()
+        return history
+    except sqlite3.OperationalError as op_e:
+        print('\nCould not perform queries into the source database: {}'
+              '\nBrowser version is not supported as it have a different sql'
+              ' schema.'.format(op_e))
```

Comment:
Not sure if the second line makes sense here - there are probably a lot of reasons why this could happen.